src/js/components/CheckBox.js

```diff
@@ -54,13 +44,15 @@ export default class CheckBox extends Component {
           </svg>
         </span>
       </span>,
-      labelNode
+      label
```

Comment:
I believe this change will affect the end result right? previously was a node with a `label` tag, and now it is a string. Any motivation for this change?